### ISSUE TITLE
[Feature] Possible option to remap keys

### ISSUE THREAD

**Issue by shadowlab** [AllenSeitz/dance-maniax-update#8]
Good evening, 

I'm looking to build myself a home cabinet using ToF sensors which can be mapped to joystick controls. Would it be possible to have key remapping options included, or even dumping to a flat file that could be manually edited with new assignments? 

Thanks for your work! Keeping the nostalgia alive for another generation 🥇

**Comment by AllenSeitz** [comment]
Yeah! I'll consider that for a new version. In the meantime you might be able to use JoyToKey to use the default keyboard controls.
